src/inet/BUILD.gn

```diff
@@ -115,6 +127,13 @@ static_library("inet") {
   }
 
   if (chip_inet_config_enable_tcp_endpoint) {
+    # Possible TCPEndpointImpl files:
+    #  - TCPEndPointImplLwIP.cpp
+    #  - TCPEndPointImplLwIP.h
+    #  - TCPEndPointImplOpenThread.cpp
```

Comment:
Above, it looks like it's possible for chip_system_config_inet to be NetworkFramework - is the config disallowed for tcp?

Reply:
Unsure here ... what I did was having a script that greps every file name as occuring in the ".gn" file and if it does not it complains.

This was the way to ensure it does not complain anymore. I believe you are likely correct, however that is to be sorted out separately. For now I plan to enforce that every file is covered by some BUILD.gn in src and will allow it to be covered by comments.